modules/ROOT/pages/choice-router-concept.adoc

```diff
@@ -85,6 +87,50 @@ http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/htt
 </mule>
 ----
 
+== Adding Additional Route Options
+
+By default, the Choice router has one route option and a default route. To add additional routes to the Choice router, follow these steps:
```

Comment:
Clarify "has one route option and a default route."
I am guessing that this means that the Choice router has a default route (determined by what or who?) and provides you the ability to specify one other route (rather than changing the default?). And you perform this task if you want to have more routes available than the usual two (default plus...one other specified by...?).

Reply:
```suggestion
By default, the Choice router has one route option and a default route. To add routes beyond these two to the Choice router, follow these steps:
```